### PR TITLE
Sidebar menu

### PR DESCRIPTION
**Pull request by VladyslavPolushyn**
from 1920px

from 769px to 1919px

to 768px

Open sidebar on small devices

### CODE REVIEW

**Review by MozdolevskyiMaksym — APPROVED** [review]

**Review by Icemandov — APPROVED** [review]

File: src/features/sidebar/sidebar.scss [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+    text-align: center;
+    font-weight: 600;
+    font-size: 18px;
+    color: white;
+
+      &:focus {
+        background-color: #88C0D0;
+        color: #2E3440;
```
Comment (Icemandov) [inline src/features/sidebar/sidebar.scss:49]: Better write like this:
```scss
background-color: map-get($colors, dark-color);
color: map-get($colors, accent-color);
```

File: src/features/sidebar/sidebar.scss [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      }
+      &:hover {
+        background-color: #88C0D0;
+        color: #2E3440;
```
Comment (Icemandov) [inline src/features/sidebar/sidebar.scss:53]: Same. Better write like this:
```scss
background-color: map-get($colors, dark-color);
color: map-get($colors, accent-color);
```

File: src/features/sidebar/sidebar.scss [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+}
+
+.sidebar--active {
+  left: 0;
+}
+
+.active {
+  background-color:#88C0D0 !important;
+  font-weight: bold !important;
+  color: #2E3440 !important;
```
Comment (Icemandov) [inline src/features/sidebar/sidebar.scss:112]: And here write like this:
```scss
background-color: map-get($colors, accent-color);
color: map-get($colors, dark-color);
```

File: src/features/sidebar/sidebar.scss (hunk 1 above)
Comment (VladyslavPolushyn) [reply src/features/sidebar/sidebar.scss:49]: Got it, thank's!

**Review by sarhan-azizov — APPROVED** [review]

**Pull request merged**